spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  let!(:post_0) { create(:post, message: "First post", id: 0) }
```

Comment:
there should be no need declare attributes for this model unless it's integral for an assertion. Let the factory handle it